app/components/header/extra_navigation_component.html.erb

```diff
@@ -1,10 +1,13 @@
 <%= tag.div(class: classes, data: { controller: "searchbox", "searchbox-search-input-id-value" => search_input_id }, role: "navigation") do %>
   <ul class="extra-navigation__list">
-    <li class="extra-navigation__link extra-navigation__mail">
-      <%= link_to("/mailinglist/signup/name", class: "link--black") do %>
-      Register your interest
-      <span class="icon icon-mail" aria-hidden="true"></span>
-      <span class="icon icon-mail-white" aria-hidden="true"></span>
+    <li class="extra-navigation__link">
+      <%= link_to(events_path({ teaching_events_search: { type: ["git"] }}), class: "link--black") do %>
```

Comment:
@ethax-ross Can you change this link to https://getintoteaching.education.gov.uk/events/about-get-into-teaching-events please - we want people to go to the showcase page, not a filtered list on the events listing page

(does this mess up your event count?